draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -50,10 +115,11 @@ export const AsyncSelect = (props: AsyncProps) => {
         NoOptionsMessage,
         SingleValue,
         MultiValue,
+        IndicatorsContainer,
         ClearIndicator: null,
         IndicatorSeparator: null,
       }}
-      className={classNames(styles.container, props.className)}
+      className={classNames(styles.specificityIncreaser, props.className)}
```

Comment:
This rename is a refactor. I was wondering why there was a `container` class wrapping everything even though it wasn't being used for any scss customisation. CSS Modules already scopes things, so it seemed unecessary. It also makes using the scss `&` operator more awkward, because you now have a grandparent selector. My guess is that it was added to increase specificity to win over react-select styles. I think this makes it more explicit why it exists.